pkg/storage/allocator.go

```diff
@@ -55,6 +55,23 @@ var (
 	// making decisions based on them.
 	// Made configurable for the sake of testing.
 	MinLeaseTransferStatsDuration = time.Minute
+
+	// EnableLeaseRebalancing controls whether lease rebalancing is enabled or
+	// not. Exported for testing.
+	EnableLeaseRebalancing = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_LEASE_REBALANCING", true)
```

Comment:
I think we should.  It's been enabled by default for a long time now.  But perhaps in a another PR.